File: core/src/main/java/com/ortin/gesturetranslator/core/managers/model_coordinate/ModelCoordinateManager.java
package com.ortin.gesturetranslator.core.managers.model_coordinate;

import com.ortin.gesturetranslator.core.managers.model_coordinate.models.ModelCoordinateArray;
import com.ortin.gesturetranslator.core.managers.model_coordinate.models.ModelCoordinateClassificationArray;

public interface ModelCoordinateManager {
    ModelCoordinateClassificationArray recognise(ModelCoordinateArray modelCoordinateArray);
}

File: core/src/main/kotlin/com/ortin/gesturetranslator/core/managers/model_coordinate/ModelCoordinateManager.kt
package com.ortin.gesturetranslator.core.managers.model_coordinate

import com.ortin.gesturetranslator.core.managers.model_coordinate.models.ModelCoordinateArray
import com.ortin.gesturetranslator.core.managers.model_coordinate.models.ModelCoordinateClassificationArray

interface ModelCoordinateManager {
    fun recognise(modelCoordinateArray: ModelCoordinateArray): ModelCoordinateClassificationArray
}
